module1/homework/calculate/calculate.hpp

```diff
@@ -2,6 +2,23 @@
 #include <string>
 
 std::string calculate(const std::string& command, int first, int second) {
-    // TODO: Implement your solution here and return proper value
-    return "";
+    std::string result = "Invalid data";
+
+    if( command == "add" ) { 
```

Comment:
please have a look at the code style guide: https://github.com/lefticus/cppbestpractices/blob/master/03-Style.md 

```suggestion
    if (command == "add") { 
```